ape_safe/exceptions.py

```diff
@@ -3,6 +3,7 @@
 
 from ape.exceptions import ApeException, ContractLogicError, SignatureError
 from ape.types import AddressType
+from requests import Response  # type: ignore[import]
```

Comment:
if you include `types-requests` in the lint dependencies, you can remove this type ignore.